uelc/main/views.py

```diff
@@ -488,14 +488,26 @@ def post(self, request):
         user.profile.profile_type = profile
         if not profile == "group_user":
             user.is_staff = True
+            self.set_image_upload_permissions(user)
         else:
             user.is_staff = False
+            self.set_image_upload_permissions(user)
```

Comment:
self.set_image_upload_permissions(user) could be moved out of the if/else statement and called just once.